erpnext/patches/v8_0/merge_student_batch_and_student_group.py

```diff
@@ -13,7 +13,15 @@ def execute():
 	frappe.reload_doctype("Student Group")
 	student_batches = frappe.db.sql('''select name as student_group_name, student_batch_name as batch,
 		program, academic_year, academic_term from `tabStudent Batch`''', as_dict=1)
+
 	for student_batch in student_batches:
+		# create student batch name if does not exists !!
+		if not frappe.db.exists("Student Batch Name", student_batch.get("batch")):
```

Comment:
if student_batch.get("batch") and not frappe.db.exists("Student Batch Name", student_batch.get("batch")):